src/Sheet/TranslationsSheetCoordinates.php

```diff
@@ -164,4 +164,14 @@ public function sourceFileColumnIndex()
     {
         return $this->getLocalesCount() + 4;
     }
+
+    public function returnRanges()
+    {
+        // Adding support for a bigger columns range fx. AA-ZZ columns
+        for ($x = 'A'; $x !== 'ZZ'; $x++) {
```

Comment:
This is super expensive, going over 26*26+26 = 702 iterations every time (plus allocating that memory), just to select one single element from the array. I would suggest using a more efficient implementation like this:

https://github.com/PHPOffice/PhpSpreadsheet/blob/36acc372875c4d894dc093825ce4f62209db5a76/src/PhpSpreadsheet/Cell/Coordinate.php#L311

Reply:
Okay I can in fact see a performance issue here with supporting the full columns range. On the other hand, the script is only run locally, so I'm not sure If it's a big deal.

 Can you submit a request with the mentioned implementation ?

Reply:
OK, I will

Reply:
Here it is: #27